syncstorage/storage/sql/queries_generic.py

```diff
@@ -251,14 +251,19 @@ def FIND_ITEMS(bso, params):
 # The idea is to delete them in small batches to keep overhead low.
 # Unfortunately there's no generic way to achieve this in SQL so the default
 # case winds up deleting all expired items.  There is a MySQL-specific
-# version using DELETE <blah> LIMIT 1000.
-PURGE_SOME_EXPIRED_ITEMS = "DELETE FROM %(bso)s "\
-                           "WHERE ttl < (UNIX_TIMESTAMP() - :grace) " \
-                           "LIMIT :maxitems"
+# version using DELETE <ttlblah> LIMIT 1000.
 
-PURGE_BATCHES = "DELETE FROM batch_uploads WHERE batch < " \
-                "   (UNIX_TIMESTAMP() - :grace) * 1000 LIMIT :maxitems"
```

Comment:
These queries were not in fact generic, I've changed them to not use the mysql-specific "LIMIT" clause.